api/serializers.py

```diff
@@ -152,6 +153,64 @@ class QuizTitleUpdateSerializer(serializers.Serializer):
     title = serializers.CharField(required=True, allow_blank=False)
 
 
+class AddQuizSessionLogSerializer(serializers.ModelSerializer):
+    quiz_session_code = serializers.CharField(required=True, allow_blank=False)
+    quiz_session_student_id = serializers.IntegerField(required=True, min_value=0)
+    question_multiple_choice_id = serializers.UUIDField(required=False)
+    action = serializers.CharField(required=True)
+
+    class Meta:
+        model = QuizSessionLog
+        fields = [
+            "quiz_session_code",
+            "quiz_session_student_id",
+            "question_multiple_choice_id",
+            "action",
+        ]
```

Comment:
You can remove this and use `serializers.Serializer` since I don't believe you're taking advantage or using anything that requires `ModelSerializer`.